src/components/badge/badge.style.ts

```diff
@@ -1,8 +1,18 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import StyledIcon from "../icon/icon.style";
-import Button from "../button";
 import Icon from "../icon";
```

Comment:
I think your current implementation uses the Button.displayName. Whilst that works I think we should avoid it and use the constructor to polymorph the element instead, that way if we change the displayName it won't break this feature